src/scratch/astrokatt/astrokatt.md

```diff
@@ -0,0 +1,292 @@
+grønt flagg. __Klikk på det!__
+
++ Du skal se at katten flytter seg rundt i en sirkel. Les koden du
+  laget en gang til. Skjønner du hvorfor katten går i sirkel?
+
++ Prøv å endre tallene i koden din. Kan du få katten til å gå saktere?
+  I større sirkler?
+
+## Sjekkliste {.check}
+
++ Nå skal vi endre litt på koden slik at vi kan kontrollere hvordan
+  katten beveger seg. I kategorien `Sansning`{.blocksensing} finnes en
+  kloss som reagerer når en tast er trykket. Den kan vi bruke til å
+  styre katten med piltastene.
+
+    Bytt ut klossene i koden din slik at den blir seende slik ut:
+
+    ```blocks
+        når grønt flagg klikkes
+        for alltid
+            hvis (tast [pil høyre v] trykket?)
+                vend høyre (5) grader
+            slutt
+            hvis (tast [pil venstre v] trykket?)
+                vend venstre (5) grader
+            slutt
+            hvis (tast [pil opp v] trykket?)
+                gå (5) steg
+            slutt
+        slutt
+    ```
+
+## Test prosjektet {.flag}
+
+__Klikk på det grønne flagget.__
+
++ Kan du styre katten rundt ved å bruke piltastene?
+
++ Forstår du hvordan katten kontrolleres?
+
++ Kan du få katten til å fly raskere eller saktere ved å endre i koden?
+
+# Steg 2: Ut i verdensrommet! {.activity}
+
+Nå skal vi sende den flyvende katten ut i verdensrommet.
+
+## Sjekkliste {.check}
+
++ Klikk på ![Velg ny bakgrunn](../bilder/velg-bakgrunn.png) nederst
+  til venstre på skjermen for å hente inn en ny bakgrunn. Velg
+  bakgrunnen `Romfart/stars`.
+
++ Vi skal også gi katten en liten oksygentank, siden den flyr rundt
+  ute i rommet. Klikk på katten i figurvinduet og deretter på fanen
+  `Drakter` øverst på skjermen.
+
++ Velg først en litt lys farge. Klikk deretter på Ellipse-verktøyet
+  til høyre på skjermen, og tegn en ellipse rundt hodet på katten.
+
+    ![](katt_oksygentank.png)
+
+Til slutt lager vi også en jordklode, som katten kan fly rundt.
+
++ Klikk på fanen `Skript` og deretter på
+  ![Velg figur fra biblioteket](../bilder/hent-fra-bibliotek.png). Velg
+  jordklodefiguren `Romfart/Earth`. Plasser denne litt på siden av skjermen.
+
+## Test prosjektet {.flag}
+
+__Klikk på det grønne flagget.__
+
++ Flyr katten rundt omkring i verdensrommet?
+
+# Steg 3: Litt mer ekte ... {.activity}
+
+Vi skal nå legge på noen effekter som gjør at spillet virker litt mer
+realistisk.
+
+## Sjekkliste {.check}
+
++ Først kan vi få jordkloden til å rotere. Dette er enkelt, vi har jo
+  allerede gjort det for katten! Pass på at jordkloden er merket i
+  figurlisten, og lag deretter dette skriptet:
+
+    ```blocks
+        når grønt flagg klikkes
+        for alltid
+            vend høyre (1) grader
+        slutt
+    ```
+
++ Videre skal vi gjøre det slik at det ser ut som om katten flyr mot
+  jorda. Det gjør vi ved å endre størrelsen slik at katten blir mindre
+  jo nærmere den kommer jordkloden.
+
+    Klikk på katten i figurlisten, og endre på skriptet ved å legge
```

Comment:
Denne setningen er lang. Hva med:

Klikk på katten i figurlisten. Legg `sett størrelse til` nederst inni `for alltid` som setter størrelsen på katten lik avstand til jorden.